meteor/lib/collections/Studios.ts

```diff
@@ -78,15 +88,30 @@ export interface DBStudio {
 
 	_rundownVersionHash: string
 
-	routeSets: {
-		[id: string]: StudioRouteSet
-	}
+	routeSets: StudioRouteSets
 
-	routeSetExclusivityGroups: {
-		[id: string]: StudioRouteSetExclusivityGroup
-	}
+	routeSetExclusivityGroups: StudioRouteSetExclusivityGroups
+
+	packageContainers: StudioPackageContainers
+	/** Which package containers is used for media previews in GUI */
+	previewContainerIds: string[]
+	thumbnailContainerIds: string[]
+}
+export interface StudioRouteSets {
```

Comment:
maybe we should be using `Record<string, StudioRouteSet>`, which could make these aliases unnecessary?
Basically the same thing but a more modern way of writing the typings